app-tracking-protection/vpn-internal/src/main/java/com/duckduckgo/vpn/internal/feature/rules/ExceptionRulesDebugReceiver.kt

```diff
@@ -112,7 +113,7 @@ class ExceptionRulesDebugReceiverRegister @Inject constructor(
         vpnStopReason: VpnStopReason,
     ) {
         // only for debug builds
-        if (appBuildConfig.isDebug) {
+        if (appBuildConfig.isDebug || appBuildConfig.isInternalBuild()) {
```

Comment:
This change is needed only if we want to revert the changes when VPN is turned OFF and back ON. This is my understanding based on the message displayed on the banner.

Remove this change if we want to keep the changes across restarts.